formats/json-tests/commonTest/src/kotlinx/serialization/features/JsonCommentsTest.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.features
+
+import kotlinx.serialization.*
+import kotlinx.serialization.json.*
+import kotlin.test.*
+
+class JsonCommentsTest: JsonTestBase() {
```

Comment:
I'm afraid we do not have particular 'main decoding cases' tests. The closes place I can think of is Json spec tests: https://github.com/Kotlin/kotlinx.serialization/blob/5e8ccad1f70a9457e0ffe6ae6b10a0bd0eaaa618/formats/json-tests/jvmTest/src/kotlinx/serialization/json/SpecConformanceTest.kt#L27

Do you think it is enough to add `allowComments = true` there?

Also, 'JsonCommentsTest' may be already enough because it has arrays, nested objects, and strings, and there's really not much anything else in the json spec.

Reply:
Ok, I think SpecConformanceTest will be enough. 
After all, this flag only changes parsing, but not the serialization logic.